aggregator/pkg/aggregator.go

```diff
@@ -285,10 +285,12 @@ func (agg *Aggregator) handleBlsAggServiceResponse(blsAggServiceResp blsagg.BlsA
 	if err == nil {
 		// In some cases, we may fail to retrieve the receipt for the transaction.
 		txHash := "Unknown"
+		effectiveGasPrice := "Unknown"
```

Comment:
Very very much a nit but in a separate pr we should move these to a constants file.